tutor/src/screens/question-library/exercise-controls.jsx

```diff
@@ -86,8 +89,15 @@ class ExerciseControls extends React.Component {
     showingDetails: PropTypes.bool,
     topScrollOffset: PropTypes.number,
     onDisplayAddEditQuestionModal: PropTypes.func.isRequired,
+    setSecondaryTopControls: PropTypes.func.isRequired,
   };
 
+  constructor(props) {
+    super(props);
+    //props.setSecondaryTopControls(this.renderControls);
```

Comment:
if this isn't needed, you can remove the entire `constructor`, however it's not clear to me where `setSecondaryTopControls` is being called from if this is removed.